[dynamo] Refactor test cross importing

jansel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #113278
* #113275
* __->__ #113242

Having tests import tests is a bit annoying because fbcode/oss have different paths.  This moves that stuff into a helper function.


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

jansel committed:
[dynamo] Refactor test cross importing

[ghstack-poisoned]

jansel commented:
@pytorchbot merge

PaliC commented:
@pytorchbot revert -m "this diff appears to be causing inductor failures internally" -c ghfirst

jansel committed:
Update on "[dynamo] Refactor test cross importing"


Having tests import tests is a bit annoying because fbcode/oss have different paths.  This moves that stuff into a helper function.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Refactor test cross importing"


Having tests import tests is a bit annoying because fbcode/oss have different paths.  This moves that stuff into a helper function.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Refactor test cross importing"


Having tests import tests is a bit annoying because fbcode/oss have different paths.  This moves that stuff into a helper function.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler

[ghstack-poisoned]

huydhn commented:
@pytorchbot revert -m 'Sorry for reverting your stack, but it is failing to list test internally with buck2' -c ghfirst

The error is https://www.internalfb.com/sandcastle/workflow/2476979795055236508
